onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -404,30 +404,50 @@ TensorrtExecutionProvider::TensorrtExecutionProvider(const TensorrtExecutionProv
     min_subgraph_size_ = std::stoi(min_subgraph_size_env);
   }
 
-  const std::string max_workspace_size_env = onnxruntime::GetEnvironmentVar(tensorrt_env_vars::kMaxWorkspaceSize);
-  if (!max_workspace_size_env.empty()) {
-    max_workspace_size_ = std::stoull(max_workspace_size_env);
+  if (info.has_trt_options) {
```

Comment:
do you think there could be scenario where user would want to control partitionIterations, minSubgGraphSize from above on per session basis?

Reply:
After the discussion with Nvidia, the five variables we added here are what Triton needed. Other variables are more like global settings. They could also be per session basis, but we haven't seen needs from users yet.